### DISCUSSION TITLE
2007/01/08/%E5%9B%9E%E7%AD%94%E4%B8%80%E4%B8%AA%E5%85%B3%E4%BA%8E-ivi-%E4%BB%AA%E5%99%A8%E9%A9%B1%E5%8A%A8%E7%A8%8B%E5%BA%8F%E7%9A%84%E9%97%AE%E9%A2%98

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#138] [Announcements]
# 2007/01/08/%E5%9B%9E%E7%AD%94%E4%B8%80%E4%B8%AA%E5%85%B3%E4%BA%8E-ivi-%E4%BB%AA%E5%99%A8%E9%A9%B1%E5%8A%A8%E7%A8%8B%E5%BA%8F%E7%9A%84%E9%97%AE%E9%A2%98

今天在我的 blog 上看到了一个关于 IVI 的问题，在这里回答一下。

https://blog.qizhen.xyz/2007/01/08/%E5%9B%9E%E7%AD%94%E4%B8%80%E4%B8%AA%E5%85%B3%E4%BA%8E-ivi-%E4%BB%AA%E5%99%A8%E9%A9%B1%E5%8A%A8%E7%A8%8B%E5%BA%8F%E7%9A%84%E9%97%AE%E9%A2%98

**Comment by ruanqizhen** [comment]
之前博客下的留言：

肖杰
2011年11月22日 16:35 编辑
请教阮老师，我是电子科大在校学生，请问有没有相关仪器互换的资料呢？我现在的方向是做仪器互换，根据IVI标准，应该就是做仪器类驱动，完全不知道如何下手，不知道您是否能提点一二？或者发点相关资料给我也行，我的邮箱jacksignal@sina.com，谢谢阮老师

回复
ruanqizhen
2011年11月24日 16:09 编辑
我有些年没有做这方面的工作了，我了解的信息也许过时了：IVI方面的资料非常少，因为做的人不多。我的bolg上可以搜到一些很陈旧的内容。估计最权威的还是参考 IVI Foundation 网站上提供的文档。他们的文档还是比较详细的。足够了解IVI开发的全过程了。

回复
Pingback： 一个 IVI 驱动程序的开发过程 « 凡人琐忆 编辑
Qizhen
2009年02月25日 09:48 编辑
你可以首先询问这家仪器公司，问他们是否提供相应的驱动程序。然后再查看一下网页 http://www.ni.com/idnet 。如果都没有，大概就是没有现成的驱动程序了。如果是在LabVIEW下使用，可以就用LabVIEW开发驱动程序。基本上调用VISA相关的函数就可以编写出驱动程序了，可以参考一下其它的驱动程序来编写。

回复
弦
2009年02月24日 17:09 编辑
请教阮先生: 如何通过Lan来实现Labview控制网络分析仪N5230A?有没有现成的驱动可供下载?从哪里可以下载?找了很多地方都下不到.如果没有现成的如何自己编写?用什么语言编写? 谢谢您!!

回复
Unknown
2007年01月20日 15:57 编辑
阮先生，谢谢您的回答，感觉稍微有点思路了。谢谢。
BTW，这个网站好难打开…

回复
